products/sql/ansible.yaml

```diff
@@ -76,15 +76,39 @@ overrides: !ruby/object:Provider::ResourceOverrides
           module.fail_json(msg="Incorrect result: {kind}".format(**result))
 
       return result
+  User: !ruby/object:Provider::Ansible::ResourceOverride
+    unwrap_resource: true
+    return_if_object: |
```

Comment:
I don't love that every object in SQL needs the same code copy/pasted here. Is there a way to factor it out somehow?